src/application/event_loop.rs

```diff
@@ -381,6 +362,74 @@ impl<T: 'static> EventLoop<T> {
               }
             };
           }
+          WindowRequest::WireUpEvents => {
+            let windows_rc = window_target.windows.clone();
+            let tx_clone = event_tx.clone();
+
+            window.connect_delete_event(move |_, _| {
+              windows_rc.borrow_mut().remove(&id);
+              if let Err(e) = tx_clone.send(Event::WindowEvent {
+                window_id: id,
+                event: WindowEvent::CloseRequested,
+              }) {
+                log::warn!("Failed to send window close event to event channel: {}", e);
+              }
+              Inhibit(false)
+            });
+
+            let tx_clone = event_tx.clone();
+            window.connect_configure_event(move |_, event| {
```

Comment:
The configure event is fired when the window is resized or changes its position but I couldn't find a way to differentiate between them so I just send both events every time

Reply:
It's fine to send both then.